src/dist/replication/meta_server/meta_bulk_load_service.h

```diff
@@ -209,12 +209,20 @@ class bulk_load_service
     /// sync bulk load states from remote storage
     /// called when service initialized or meta server leader switch
     ///
-    void create_bulk_load_root_dir(error_code &err, task_tracker &tracker);
+    void create_bulk_load_root_dir();
 
-    void sync_apps_bulk_load_from_remote_stroage(error_code &err, task_tracker &tracker);
+    void sync_apps_bulk_load_from_remote_storage();
+
+    void do_sync_app_bulk_load(int32_t app_id);
+
+    void sync_partitions_bulk_load_from_remote_storage(int32_t app_id, const std::string &app_name);
+
+    void do_sync_partition_bulk_load(const gpid &pid,
```

Comment:
Most of your function names are designed very verbose. For example, it's weird for `bulk_load_service::on_sync_partition_bulk_load` to repeat the word 'bulk_load'. Naming it `bulk_load_service::on_sync_partition` doesn't make it less readable. You can refactor the name in the later PR if possible.

Reply:
I think it is okay for long function name if it is readable for who are reading the code. `on_sync_partition_bulk_load` is used to sync `partition_bulk_load_info` structure from remote storage, I will simplify those functions in further pull request.

Reply:
I think longer function means more efforts to read them. It depends on case. If `on_sync_partition_bulk_load` was a function of `meta_service`, it's fine to give it a long name.

Reply:
I have already updated function name.